system-addon/content-src/components/Card/Card.jsx

```diff
@@ -56,14 +56,18 @@ class Card extends React.Component {
     const isContextMenuOpen = this.state.showContextMenu && this.state.activeCard === index;
     // Display "now" as "trending" until we have new strings #3402
     const {icon, intlID} = cardContextTypes[link.type === "now" ? "trending" : link.type] || {};
+    const hasImage = link.image || link.imageLoading;
+    const imageStyle = {backgroundImage: link.image ? `url(${link.image})` : "none"};
 
     return (<li className={`card-outer${isContextMenuOpen ? " active" : ""}${props.placeholder ? " placeholder" : ""}`}>
       <a href={link.url} onClick={!props.placeholder && this.onLinkClick}>
         <div className="card">
-          {link.image && <div className="card-preview-image" style={{backgroundImage: `url(${link.image})`}} />}
-          <div className={`card-details${link.image ? "" : " no-image"}`}>
+          {hasImage && <div className="card-preview-image-outer">
+            <div className={`card-preview-image${link.image ? " active" : ""}`} style={imageStyle} />
```

Comment:
The use of "active" here is confusing as at least with css-related stuff, it's when the user is interacting with it. Probably name this "loaded".